mapclassify/_classify_API.py

```diff
@@ -33,6 +34,7 @@
     "naturalbreaks": NaturalBreaks,
     "quantiles": Quantiles,
     "percentiles": Percentiles,
+    "pretty": PrettyBreaks,
```

Comment:
This is the only one where key doesn't match lowercase class. Should this be `"prettybreaks": PrettyBreaks` or should the class be called `Pretty` only if we want to use `pretty` as a key?